helpdesk_mgmt/views/helpdesk_ticket_templates.xml

```diff
@@ -483,16 +483,14 @@
                         for="attachment"
                     >Add Attachments</label>
                     <div class="col-md-7 col-sm-8">
-                        <div class="btn btn-default btn-file col-md-12">
```

Comment:
Why removing the div?

Reply:
A mistake on my part, I'm restoring it now